postgres-appliance/scripts/post_init.sh

```diff
@@ -137,6 +137,11 @@ while IFS= read -r db_name; do
     if [ "x$TIMESCALEDB_VERSION" != "x" ] && [ "x$TIMESCALEDB_VERSION" != "x$TIMESCALEDB" ]; then
         echo "ALTER EXTENSION timescaledb UPDATE;"
     fi
+    POSTGISDB_VERSION=$(echo -e "SELECT NULL;\nSELECT extversion FROM pg_extension WHERE extname = 'postgis'" | psql -tAX -d ${db_name} 2> /dev/null | tail -n 1)
```

Comment:
There are multiple issues.
1. `POSTGIS_VERSION` environment variable is not available here.
2. we probably need to run `ALTER EXTENSION` not only for `postgis`